### PR TITLE
New docs: What is Fuel, Why Fuel

### PR DESCRIPTION
**Pull request by camiinthisthang**
New sections added:

- What is Fuel
- Why Fuel

When I run the linter on it locally, I get errors about line length not being what it wants. Not sure what this is about 

> src/what-is-fuel.md:3:81 MD013/line-length Line length [Expected: 80; Actual: 166]

### CODE REVIEW

**Comment by camiinthisthang** [comment]
@adlerjohn I set the PR to merge into master, but should I have set it to the iA branch? I assumed that'll get merged soon so I figured to set base branch as master. Lmk if I should change or if this is fine.

**Comment by adlerjohn** [comment]
You can re-target it against the branch in #1, then once that's merged github will actually automatically re-target it against `master`.

**Review by adlerjohn — COMMENTED** [review]
merge conflicts

File: src/what-is-fuel.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Sway Language
+
+Sway is a domain-specific language (DSL) for the Fuel Virtual Machine (FuelVM), a blockchain-optimized VM designed for the Fuel blockchain. Sway is based on Rust and includes syntax to leverage a blockchain VM without a needlessly verbose boilerplate.
+
+Sway was created alongside the FuelVM and designed for the high-compute Fuel environment. Check out the Sway docs here.
```
Comment (otrho) [inline src/what-is-fuel.md:45]: Where?

**Review by adlerjohn — COMMENTED** [review]
Hmmm. So, the two pages seem to have a lot of overlap. E.g. the intro repeats the exact same three points, and the contents also explain the same three points. Is there some way of either further differentiating the two, or if not, just merging them?

**Comment by camiinthisthang** [comment]
> Hmmm. So, the two pages seem to have a lot of overlap. E.g. the intro repeats the exact same three points, and the contents also explain the same three points. Is there some way of either further differentiating the two, or if not, just merging them?

Going to split this into 2 PRs so we can merge What is Fuel, and then will rewrite Why fuel. I agree, they are too similar, and after thinking about it for a few days I have a better idea on how to approach Why fuel section.

**Pull request closed without merging**